tests/unitTests/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-add_executable(unitTests "")
+add_executable(unitTests "" boundarySolver/physics/logLawBoundaryTests.cpp)
```

Comment:
can you remove this if it already added to the other cmakeLists?

Reply:
Not intentional. Must have done something wrong.